db/migrate/20170620132731_create_budget_investment_milestones.rb

```diff
@@ -0,0 +1,11 @@
+class CreateBudgetInvestmentMilestones < ActiveRecord::Migration
+  def change
+    create_table :budget_investment_milestones do |t|
+      t.integer :investment_id
+      t.string   "title", limit: 80
```

Comment:
We're already checking the title's presence on the model, but I'm always a fan of also setting database restrictions ( a `null: false` in this case) but no worries, can be added later on or even we can live without it I think